### PR TITLE
Update guestbook.md

### PR DESCRIPTION
**Pull request by guzj11**
sync with en version.

### CODE REVIEW

File: content/zh/docs/tutorials/stateless-application/guestbook.md [hunk 1]
```diff
@@ -615,12 +615,14 @@ Deleting the Deployments and Services also deletes any running Pods. Use labels
...
 
 <!--
+* Add [ELK logging and monitoring](/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk/) to your Guestbook application
 * Complete the [Kubernetes Basics](/docs/tutorials/kubernetes-basics/) Interactive Tutorials
 * Use Kubernetes to create a blog using [Persistent Volumes for MySQL and Wordpress](/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/#visit-your-new-wordpress-blog)
 * Read more about [connecting applications](/docs/concepts/services-networking/connect-applications-service/)
 * Read more about [Managing Resources](/docs/concepts/cluster-administration/manage-deployment/#using-labels-effectively)
 -->
 
+* 添加 [ELK 日志与监控] (/zh/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk/)到Guestbook应用。
```
Comment (tengqm) [inline content/zh/docs/tutorials/stateless-application/guestbook.md:625]: 这里的额外空格会导致链接失效。
Guestbook 前后添加空格。
此行过长，请考虑折行。

File: content/zh/docs/tutorials/stateless-application/guestbook.md [hunk 2]
```diff
@@ -615,12 +615,14 @@ Deleting the Deployments and Services also deletes any running Pods. Use labels
...
 
 <!--
+* Add [ELK logging and monitoring](/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk/) to your Guestbook application
 * Complete the [Kubernetes Basics](/docs/tutorials/kubernetes-basics/) Interactive Tutorials
 * Use Kubernetes to create a blog using [Persistent Volumes for MySQL and Wordpress](/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/#visit-your-new-wordpress-blog)
 * Read more about [connecting applications](/docs/concepts/services-networking/connect-applications-service/)
 * Read more about [Managing Resources](/docs/concepts/cluster-administration/manage-deployment/#using-labels-effectively)
 -->
 
+* 添加[ELK 日志与监控](/zh/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk/)到 Guestbook 应用
```
Comment (tengqm) [inline content/zh/docs/tutorials/stateless-application/guestbook.md:625]:
```suggestion
* 添加 [ELK 日志与监控](/zh/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk/)
  到 Guestbook 应用
```

**Comment by tengqm** [comment]
/approve cancel

Two problems.

- There are 3 commits. Please squash.
- The commit message in your original commit is invalid, please fix.

**Comment by guzj11** [comment]
> 
> 
> /approve cancel
> 
> Two problems.
> 
>     * There are 3 commits. Please squash.
> 
>     * The commit message in your original commit is invalid, please fix.

Fixed

**Pull request merged**